### PR TITLE
Bump latest versions

### PR DESCRIPTION
**Pull request by brylie**
This is a BinderHub version bump. See the link below for a diff of new changes:

https://github.com/jupyterhub/binderhub/compare/7b2c4f8...5536a0f

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ mybinder/requirements.lock
 
 docs/_build
 travis/crypt-key
+env
```
Comment (betatim) [inline .gitignore]: What is in env? And how did it get here?

**Comment by betatim** [comment]
Looks good to me, just one question about how/why there is a new file we need to ignore.

**Comment by brylie** [comment]
I needed to install the requirements.txt, and didn't want to do so in my global python environment. So, I created a local environment in the `env` folder with `python3 -m venv env`.

Similar examples can be seen in the [GitHub Python .gitignore](https://github.com/github/gitignore/blob/master/Python.gitignore#L98-L105).

**Comment by betatim** [comment]
Thanks! Deploying now.

**Comment by choldgraf** [comment]
wooo, thanks @brylie !!

**Comment by meeseeksmachine** [comment]
This pull request has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/mybinder-org-upgrades-and-updates/712/11

**Pull request merged**